mars/sentence_definition_scoring.py

```diff
@@ -0,0 +1,85 @@
+import mars.db
+from mars import logging
+from mars.db import collections
+from mars.db.db_fields import (
+    ID,
+    SENTENCE,
+    SENTENCE_DOC_ID,
+    IS_DEFINITION
+)
+from mars.definition_model import DistilBertBaseUncased
+from typing import List
+BATCH_SIZE = 100
+
+logger = logging.new_logger(__name__)
+
+
+def get_docs_range(key_min: int, key_max: int) -> List[str]:
+    res = []
+    for i in range(key_min, key_max + 1):
+        res.append("Documents/" + str(i))
+    return res
+
+
+def document_definition_scoring(key_min: int, key_max: int, path_to_model:str = "../models/distilbert-base-uncased") -> None:
+    """
+
+    @param key_min: lowest document key
+    @param key_max: highest document key
+    @param path_to_model: path to folder with definition model
+    """
+    get_docs_query = f"FOR u IN {collections.DOCUMENTS} " \
+                     f"FILTER TO_NUMBER(u._key) >= {key_min} && TO_NUMBER(u._key) <= {key_max}" \
+                     f" RETURN DISTINCT u.{ID}"
+
+    all_docs = mars.db.database.AQLQuery(get_docs_query, 10000, rawResults=True)
+
+    # get all sentences from docs
+    get_done_docs_query = f"FOR u IN {collections.SENTENCES} " \
+                          f"FILTER u.{SENTENCE_DOC_ID} IN TO_ARRAY({get_docs_range(key_min, key_max)}) " \
```

Comment:
Idea przedziału była taka, żeby nie wrzucać do stringa query całej listy id.
Zamień to na nierówności jak wyżej

Reply:
No tutaj też filtruję, tylko jako że ID było inne - `Documents/<jakieś_id>` to trzeba było inaczej robić, ale `get_docs_range` robi nierówności

Reply:
No można, to prawda